src/renderer/log/log.css

```diff
@@ -1,27 +1,25 @@
 /* Defined styles for Log.js */
 
-.logContainer {
-  height: 100%;
-  display: table;
-}
-
-.logContainer pre {
-  font-family: Menlo, monospace;
+.logContainer .row {
+  display: flex;
   margin: 0;
-  display: block;
-  overflow-y: auto;
 }
 
-.messages {
-  background-color: #eee;
+.logContainer .messages {
+  background-color: #000;
```

Comment:
honestly, not a fan of the black log background... but I guess if we're going with a dark scheme...

Maybe make it a little off-black? like #141414? or something?

Reply:
Resolved in https://github.com/NGCP/GCS/commit/7281b09d05d28cd92286e87185713d2b03d56da1